runtime/doc/help.txt

```diff
@@ -167,6 +167,8 @@ DEVELOPING NVIM
 
 |dev|			Development of Nvim
 |dev-style|		Development style guidelines
+|dev-theme|		Theming style guidelines
```

Comment:
I've already added that in the faq.txt PR; not a blocker, but maybe check it doesn't conflict?

Reply:
Or let me just merge that PR, and you can rebase?

Reply:
Sure, let't do that

Reply:
Nite: the theme guidelines are design guidelines, not style guidelines.

Reply:
Go ahead and rebase (but pull first!)